packages/analytics-core/src/config.ts

```diff
@@ -30,6 +32,7 @@ export const getDefaultConfig = () => ({
   serverUrl: AMPLITUDE_SERVER_URL,
   serverZone: 'US' as ServerZoneType,
   useBatch: false,
+  diagnosticProvider: new BaseDiagnostic(),
```

Comment:
Can we disable this for non-Browser usage?

Reply:
Change the default value of `isDisable` to true in BaseDiagnostic in https://github.com/amplitude/Amplitude-TypeScript/pull/568/commits/80a811483b2725fc81124966fb664c6640f5c37c. So leave here as it is.